### PR TITLE
Use k8s_client to replace the Client in validate_job_status.py.

### PR DESCRIPTION
**Pull request by workingloong**
The Client in validate_job_status.py has code duplication with `k8s_client.Client` and deviation from DRY (Do not repeat yourself) principle.

### CODE REVIEW

File: scripts/setup_k8s_env.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+sudo pip3 install tensorflow
```
Comment (terrytangyuan) [inline scripts/setup_k8s_env.sh:3]: Why is this needed for status validation?

Comment (workingloong) [reply scripts/setup_k8s_env.sh:3]: Because   the module https://github.com/sql-machine-learning/elasticdl/blob/492fe94a377ff15cb67468fefbc99f89a16147a2/elasticdl/python/common/k8s_client.py#L13 in `elasticdl.python.common.k8s_client.Client`  imports Tensorflow and I import `elasticdl.python.common.k8s_client.Client` in `validate_job_status.py`.

Comment (workingloong) [reply scripts/setup_k8s_env.sh:3]: I find dependency packages for `elasticdl.python.common.k8s_client.Client` is too many. Tensorflow, odps and others. So, it is more simple to use an indenpent `Client` for `validate_job_status.py`. I will close the PR.
